src/game/Ship.java

```diff
@@ -209,6 +226,10 @@ public void setSystems(int id, double value) {
 
     }
 
+    public double getMaxSystems(int id) {
```

Comment:
We wcześniejszym komentarzu zaproponowałem zmianę implementacji przechowywania maksymalnych wartości dla systemów (zamiast lista maksów, każda zmienna miałaby własne pole min i maks dzięki klasie `GenomeAttribute`), ale tak czysto formalnie: jak mamy listę `maxSystems`, to metoda `getMaxSystems` powinna zwracać tę listę (albo jej kopię jeśli lista ma być immutable). Jak chcesz zwracać konkretną wartość z tej listy, to lepiej nazwać `getMaxSystem` albo idealnie `getMaxSystemById` i w metodzie sprawdzać np. czy podane id nie jest za duże/mniejsze niż zero, i rzucać `IllegalArgumentException`.